regolith/mongoclient.py

```diff
@@ -314,6 +318,8 @@ def insert_one(self, dbname, collname, doc):
     def insert_many(self, dbname, collname, docs):
         """Inserts many documents into a database/collection."""
         coll = self.client[dbname][collname]
+        for doc in docs:
+            doc['_id'].replace('.', '')
```

Comment:
The reason I know that this bit of code is necessary is because it is used in the conftest.py code when creating the mongo test fixture. There actually are _ids in the schema exemplars with dots, but they get removed in the creation of the test fixture.

If we want to test them, I'll just take that out.

Whenever I wrote the test initially I must have been more focused on making the test pass than making sure the code worked. Big wupsie daisy. 

Not sure that we need a decorator since this is specific to mongo backend collection entry, so it shouldn't actually be repeated that often. Let me know why you are thinking decorator. Maybe at morning meeting.